Special abilities PR 5, Split Speed values (#8)
GameBrains/er-core: modules/special-rules/pages/firepower-1.adoc

= Firepower +1

> These units are armed with potent weaponry that does not affect their range.

Reviewer: ```suggestion
These units are especially potent at range for their kind.
```
